src/components/Dialog/Dialog.stories.tsx

```diff
@@ -160,10 +160,23 @@ Message_Dialog.parameters = {
 export const Action_Dialog: Story = () => {
   const [isOpen, setIsOpen] = useState(false)
   const [value, setValue] = React.useState('Apple')
+  const [responseMessage, setResponseMessage] = useState<{
+    status: 'success' | 'error' | 'processing'
+    text: string
+  }>()
   const onClickOpen = () => setIsOpen(true)
   const onClickClose = () => setIsOpen(false)
```

Comment:
It is better that reset responseMessage state when actioned or closed.
Because we have to reset state in real use cases and code in the story are good example.

実際に使用するときは閉じる時やActionを行ったときに responseMessage state をリセットすると思うので、Storyでもそうしておくと良さそう。Storyのコードが良い例になるし。

Reply:
onClickClose / onClickAction それぞれで responseMessage を初期化するように修正